src/learn/qml/MuseScore/Learn/internal/ClassesPage.qml

```diff
@@ -63,141 +63,148 @@ FocusScope {
         }
     }
 
-    Rectangle {
-        anchors.top: parent.top
-        anchors.left: parent.left
-        anchors.leftMargin: root.sideMargin
-        anchors.right: parent.right
-        anchors.rightMargin: root.sideMargin
+    ScrollView {
+        ScrollBar.horizontal.policy: ScrollBar.AlwaysOff
+        anchors.fill: parent
+        clip: true
```

Comment:
Thanks for the patch! It is preferred to use our `StyledFlickable` item instead:
```suggestion
    StyledFlickable {
        anchors.fill: parent
```

Also, on some other scrollable pages, we add a subtle gradient at the top and bottom of the scrollable area. I think that would be nice here too. See `Plugins/PluginsPage.qml` and `ScoresPage.qml` for inspiration.